subsys/usb/usb_c/usbc_tc_src_states.c

```diff
@@ -0,0 +1,333 @@
+/*
+ * Copyright (c) 2022 The Chromium OS Authors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_DECLARE(usbc_stack, CONFIG_USBC_STACK_LOG_LEVEL);
+
+#include "usbc_stack.h"
+#include "usbc_tc_src_states_internal.h"
+
+/**
+ * @brief 4.5.2.2.7 Unattached.SRC State
```

Comment:
Nit: I'd cite the spec version in case they change the section numbering later.

Reply:
Done